.changeset/dull-carpets-breathe.md

```diff
@@ -21,10 +21,10 @@ import ReactComponent from "../components/button.jsx"
 import VueComponent from "../components/button.vue"
 
 // MDX runtime is contained inside the Astro
-import mdxRenderer from "@astrojs/jsx/serverr.js"
+import mdxRenderer from "@astrojs/jsx/server.js"
```

Comment:
Oh. This should be `astro/jsx/server.js`? We don't have a `@astrojs/jsx` package?

Reply:
I think it may be an internal package so people usually dont have to install it?

Reply:
@florian-lefebvre Nope, this is a misspelled package name. It's `astro/jsx/server.js` indeed.

Reply:
Another typo of mine, sorry. Probably a bug. It should be `astro/jsx/server.js`